### PR TITLE
Faq page

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: coss/home/content_blocks.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from wagtail.wagtailcore import blocks
+
+
+class InputTextContentBlock(blocks.StructBlock):
+    """Reusable component which adds a title followed by some text."""
+
+    title = blocks.CharBlock(max_length=150)
+    text = blocks.RichTextBlock(max_length=1000)
```
Comment (comzeradd) [inline coss/home/content_blocks.py]: Do you think it makes sense to have a link field here too and have all these fields as non-mandatory? In order to have one content block for all similar cases. Or should we create multiple with different required fields? (eg. title, link)

Comment (akatsoulas) [reply coss/home/content_blocks.py]: 👍 it's a good idea to handle all input in a block. I will update the PR.

**Comment by akatsoulas** [comment]
@comzeradd r? PR should be complete now.

**Comment by akatsoulas** [comment]
@comzeradd pr updated

**Pull request merged**